reflectometry/amor_reduction_detailed.ipynb

```diff
@@ -63,7 +75,12 @@
     "x = data.coords['position'].fields.x.copy()\n",
     "data.coords['position'].fields.x = data.coords['position'].fields.z\n",
     "data.coords['position'].fields.z = data.coords['position'].fields.y\n",
-    "data.coords['position'].fields.y = x"
+    "data.coords['position'].fields.y = x\n",
+    "# Amor records event time-of-flight in nanoseconds,\n",
+    "# but for convenience we convert this to the more commonly used unit of microseconds.\n",
+    "data.bins.coords['tof'] = sc.to_unit(data.bins.coords['tof'].astype('float64'), 'us')\n",
```

Comment:
why is the `astype` necessary?

Reply:
Because they are integer nanoseconds and if you simply convert to microseconds you loose precision (i think?)

Reply:
Yes - As a separate issue, maybe this could better be incorporated into `sc.to_unit` provided that copy=True. i.e. when your length scale is increasing your operations involving integers could result in truncation and should therefore be done in double precision maybe.

Reply:
I think `to_unit` would then have to be a bit too smart and do things behind the scenes. I think I prefer it as it is now.